src/plugins/intel_gpu/src/kernel_selector/cl_kernels/mvn_gpu_bfyx_opt.cl

```diff
@@ -54,23 +52,14 @@ KERNEL (mvn_gpu_bfyx_opt)(
         my_sum += (float)input[data_set_offset + workers_per_data_set * items_num + in_data_set_idx];
     }
 
-    lg_storage[in_data_set_idx] = my_sum;
-
-    barrier(CLK_LOCAL_MEM_FENCE);
-    for (uint offset = workers_per_data_set / 2; offset > 0; offset /= 2) {
-        if (in_data_set_idx < offset) {
-            lg_storage[in_data_set_idx] += lg_storage[in_data_set_idx + offset];
-	}
-        barrier(CLK_LOCAL_MEM_FENCE);
-    }
+    my_sum = work_group_reduce_add(my_sum);
 
     if (in_data_set_idx == 0)
     {
-        lg_storage[0] /= data_set_size;
+        my_sum /= data_set_size;
     }
-    barrier(CLK_LOCAL_MEM_FENCE);
 
-    my_sum = lg_storage[0];
+    my_sum = work_group_broadcast(my_sum, 0);
```

Comment:
Do we need this broadcast? Can't each WI just divide its local `my_sum` by `data_set_size`

Reply:
I think it turned out to be faster this way, will check again